docs/user-guides/devcontainers/index.md

```diff
@@ -50,32 +45,25 @@ When a workspace with the dev containers integration starts:
 
 - Automatic dev container detection from repositories
 - Seamless dev container startup during workspace initialization
+- Dev container change detection and dirty state indicators
+- On-demand dev container recreation via rebuild button
 - Integrated IDE experience in dev containers with VS Code
 - Direct service access in dev containers
-- Limited SSH access to dev containers
+- SSH access to dev containers
+- Automatic port detection for container ports
 
-### Coming Soon
+## Limitations
 
-- Dev container change detection
-- On-demand dev container recreation
-- Support for automatic port forwarding inside the container
-- Full native SSH support to dev containers
-
-## Limitations during Early Access
-
-During the early access phase, the dev containers integration has the following
-limitations:
+The dev containers integration has the following limitations:
 
+- **Not supported on Windows** - Dev containers require a Linux or macOS workspace
```

Comment:
```suggestion
- Not supported on Windows
```
Mainly, I want the AI dash gone, but IMO we don't need to many repeats of the same information.